integrationExamples/gpt/haloRtdProvider_example.html

```diff
@@ -113,18 +113,9 @@
             googletag.pubads().disableInitialLoad();
         });
 
-        var bidSegmentMappers = {
-            appnexus: function(bid, segments) {
-                if (!bid.params) {
-                    bid.params = {}
-                }
-                if (!bid.params.user) {
-                    bid.params.user = {}
-                }
```

Comment:
Shouldn't this stay in the example to show the segment mappings working? I would suggest putting this back for appnexus and add another bidder that uses the 'generic' mapping defined in your rtd module.

Reply:
The generic mapper is used for testing or if a bidder wants to pull from the taxonomy group's generic taxonomy.  Both the appnexus and generic mapper are included in the rtd module and enabled by passing 'true' to the segment mapper config.

Reply:
We would want still want to show in the example that you can override what is included in the rtdmodule with a custom function for appnexus, since this is the major feature of this PR.